src/entities/exportAccount.ts

```
@@ -0,0 +1,139 @@
+      actorProfile: serializeAccount(
+        { ...account, successor: null },
+        c.req.url,
+      ),
+      outbox: serializedPosts,
+      lists: serializedLists,
+      followers: serializedFollowers,
+      followingAccounts: serializedFollowing,
+      bookmarks: serializedBookmarks,
+    });
+
+    return c.body(exportTarballStream as any, 200, {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)